### PR TITLE
Update @types package versions in package.json from dependabot updates to yarn.lock

### PR DESCRIPTION
**Pull request by dannyvv**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6955)

### CODE REVIEW

**Comment by dannyvv** [comment]
This change is a manual experiment. Considering writing a tool to automate this for cases where yarn.lock resolves higher version than we have manually declared in package.json.

File: packages/react-native-platform-override/src/e2etest/collateral/0.62.2/flowconfig.windows.conflict [hunk 1]
```diff
@@ -87,6 +87,8 @@ untyped-type-import
 [version]
 <<<<<<< Upstream
 ^0.113.0
+||||||| base
```
Comment (dannyvv) [inline packages/react-native-platform-override/src/e2etest/collateral/0.62.2/flowconfig.windows.conflict:90]: I'm not sure if our tests are affected by local state on my machine or if @types files can have an influence on the diff we generate. I ran into this change when bumping versons for the appium replacement too.

Comment (dannyvv) [reply packages/react-native-platform-override/src/e2etest/collateral/0.62.2/flowconfig.windows.conflict]: And it doesn't reply on the mac... Will investigate what state of my devbox is leaking into the tests....

Comment (dannyvv) [reply packages/react-native-platform-override/src/e2etest/collateral/0.62.2/flowconfig.windows.conflict]: hmm, it just uses git diff so it relies on local git config :(

Comment (dannyvv) [reply packages/react-native-platform-override/src/e2etest/collateral/0.62.2/flowconfig.windows.conflict]: too many options to git diff and can't find anything in `git config --list` that might cause this... I'll just accept my tests don'r pass locally...

**Review by NickGerleman — APPROVED** [review]

**Pull request merged**